ja/splatoon2.json

```diff
@@ -124,7 +123,7 @@
             "B17": "タタキケンサキ",
             "B97": "クマサン商会",
             "B98": "アタリメイド",
-            "B99": "amiibo"
```

Comment:
Do we have confirmation that this is what's shown in-game? Both the splatoon2 and splatoon3 source language files have this written as 'amiibo'

Reply:
From https://leanny.github.io/data/Languages/lang_dict_JPja.json:
![image](https://user-images.githubusercontent.com/16487249/188520236-b8691a82-8ff1-4286-9e8a-b7c7715b5033.png)

Reply:
This is indeed what is what is in the game (from splatoon 2 footage on YouTube). Therefore the amiibo string was untranslated. (although I think it would be fine to just use アミーボ instead of amiibo)